packages/editor-bricks-helper/src/internal/getAppendingNodesAndEdges.ts

```diff
@@ -43,11 +45,15 @@ export function getAppendingNodesAndEdges(
 
     if (
       isBrickNode(builderNode) &&
-      !builderNode.brick.includes(".") &&
-      builderNode.brick.startsWith("tpl-") &&
-      !processedTemplateSet.has(builderNode.brick) &&
-      (templateSource = templateSourceMap?.get(builderNode.brick)) &&
-      templateSource.children?.length > 0
+      ((!builderNode.brick.includes(".") &&
+        builderNode.brick.startsWith("tpl-") &&
+        !processedTemplateSet.has(builderNode.brick) &&
+        (templateSource = templateSourceMap?.get(builderNode.brick)) &&
+        templateSource.children?.length > 0) ||
+        ((templateSource = storyList.find(
```

Comment:
1. `storyList` 应该是异步拿的，可能那时还没有，要加个 `?`；
2. 我们这里限定下只针对 custom template，因为 storyList 列表很大。

```suggestion
        ( builderNode.brick.includes(".tpl-") && (templateSource = storyList?.find(
```